modules/caddyhttp/app.go

```diff
@@ -346,30 +346,24 @@ func (app *App) Start() error {
 					tlsCfg := srv.TLSConnPolicies.TLSConfig(app.ctx)
 					ln = tls.NewListener(ln, tlsCfg)
 
-					/////////
-					// TODO: HTTP/3 support is experimental for now
-					if srv.ExperimentalHTTP3 {
```

Comment:
Shouldn't we just add a new option `DisableHTTP3` and flip the boolean here?

Reply:
Don't we already have facilities to configure protocols? It might be ALPN, and HTTP/3 supports the "h3" identifier in ALPN: https://quicwg.org/base-drafts/draft-ietf-quic-http.html#section-3.1